classes/services/PKPSubmissionService.inc.php

```diff
@@ -727,15 +773,13 @@ public function getPropertyStages($submission, $stageIds = null) {
 
 				case WORKFLOW_STAGE_ID_INTERNAL_REVIEW:
 				case WORKFLOW_STAGE_ID_EXTERNAL_REVIEW:
-					import('lib.pkp.classes.submission.reviewRound.ReviewRoundDAO');
 					$reviewRoundDao = DAORegistry::getDAO('ReviewRoundDAO');
 					$reviewRound = $reviewRoundDao->getLastReviewRoundBySubmissionId($submission->getId(), $stageId);
 					if ($reviewRound) {
 						$stage['statusId'] = $reviewRound->determineStatus();
 						$stage['status'] = __($reviewRound->getStatusKey());
 
 						// Revision files in this round.
-						import('lib.pkp.classes.submission.SubmissionFile'); // Import constants
```

Comment:
Beware of removing imports like this if you haven't investigated it pretty thoroughly -- the SubmissionFile import in particular is sometimes needed to load `SUBMISSION_FILE_`... constants, which in some conditions might not be loaded yet.

Reply:
D'oh! I've restored all of them just in case.

Reply:
Yes, it is needed for the lines that come after this one... -- I am just testing...